autoload/pandoc/hypertext.vim

```diff
@@ -270,7 +270,7 @@ function! pandoc#hypertext#OpenSystem(...) abort
     endif
 
     if has('unix') && executable('xdg-open')
-        call system('xdg-open '. shellescape(expand(addr,':p')))
```

Comment:
maybe mistaking `expand()` for `fnamemodify()`
`expand()` expands globs and env-var so this may cause something unexpected
otherwise just `addr` is returned